CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 ## HEAD
 
+## 0.15.0
+###### _May 5, 2016_
+
+Please read through the aplha and beta releases from 0.14.4 too as their changes are not listed here.
```

Comment:
What I meant to say was the alphas and betas after the 0.14.4 I guess it's confusing O.o I'll reword :grin: